src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -75,11 +75,11 @@ type AttributeConfig = BaseAttributeConfig | BooleanAttributeConfig
 
 export interface Attributes {
   id: string
-  host: string | undefined
+  host?: string | undefined
```

Comment:
If its optional I think the undefined type is not needed. Minor thing...

Reply:
good catch, that shouldn't accept `undefined` anymore, thx